cypress/fixtures/views/start-with-step-by-step.html

```diff
@@ -9,13 +9,6 @@
   {{ govukHeader() }}
 {% endblock %}
 
-{% block pageScripts %}
-  <script type="text/javascript">
-    var $element = document.querySelector('#step-by-step-navigation')
-    var stepByStepNavigation = new GOVUK.Modules.AppStepNav($element).init()
-  </script>
-{% endblock %}
-
```

Comment:
How come this isn't needed any more? Do we need to remove the block from the examples in the v12 support branch as well?

Reply:
I want to revisit this - this script should really be in `@govuk-prototype-kit/step-by-step` and added through the extensions framework.  I'm not sure if it currently is.

Reply:
How can the tests pass if it doesn't? 🤔

Reply:
I see. The tests pass because we're installing v1 of the extension, which does autoload. If we were using v2 these lines would be necessary, because they aren't included in the package.